packages/Button/stories/Button.stories.styles.js

```diff
@@ -12,4 +13,11 @@ export const ButtonStory = styled(Story)`
       margin-left: ${tokens.space};
     }
   }
+  a,
```

Comment:
You just need to add `a, button, ... { margin-left: ${tokens.space}; }`. You don't need an additional block that has the same css rule.

Reply:
done